dependencies.props

```diff
@@ -9,30 +9,29 @@
        These ref versions are pulled from https://github.com/dotnet/versions.
   -->
   <PropertyGroup>
-    <CoreFxCurrentRef>40697681be31dff83b6c3b75a6de0d3adbc790a7</CoreFxCurrentRef>
-    <CoreClrCurrentRef>3a037b925fac2dfab6ca2f587099b45bb8413f5f</CoreClrCurrentRef>
+    <CoreFxCurrentRef>9bd62fa21bf4815effc5aa1906e32d4680a16a57</CoreFxCurrentRef>
+    <CoreClrCurrentRef>5cce8ebd18593f26147fcd04eb7183ae7fb87358</CoreClrCurrentRef>
     <CoreSetupCurrentRef>40697681be31dff83b6c3b75a6de0d3adbc790a7</CoreSetupCurrentRef>
     <ExternalCurrentRef>1fe3c36279b702b526ca9441412d441f1b6a4821</ExternalCurrentRef>
-    <ProjectNTfsCurrentRef>40697681be31dff83b6c3b75a6de0d3adbc790a7</ProjectNTfsCurrentRef>
-    <ProjectNTfsTestILCCurrentRef>40697681be31dff83b6c3b75a6de0d3adbc790a7</ProjectNTfsTestILCCurrentRef>
+    <ProjectNTfsCurrentRef>62bbc1ed21d32925454339e1bb0a17590029c9d4</ProjectNTfsCurrentRef>
+    <ProjectNTfsTestILCCurrentRef>d55da128c5ce03e979a3c2e3f5ecd91da24e259c</ProjectNTfsTestILCCurrentRef>
     <SniCurrentRef>97059fa979a3c8fb8b9fba127c526f15e48c9dde</SniCurrentRef>
     <StandardCurrentRef>40697681be31dff83b6c3b75a6de0d3adbc790a7</StandardCurrentRef>
   </PropertyGroup>
 
   <!-- Auto-upgraded properties for each build info dependency. -->
   <PropertyGroup>
-    <PlatformPackageVersion>2.1.0-preview1-25631-01</PlatformPackageVersion>
-    <CoreFxExpectedPrerelease>preview1-25801-01</CoreFxExpectedPrerelease>
-    <CoreClrPackageVersion>2.1.0-b-uwp6-25707-02</CoreClrPackageVersion>
+    <CoreFxExpectedPrerelease>preview1-26307-01</CoreFxExpectedPrerelease>
```

Comment:
I'm not sure if this is correct or not? Should the CoreFX dependency be a uwp label as well?

Reply:
Packaging.props currently has `<PreReleaseLabel>preview1</PreReleaseLabel>`. What value would you like to see? I can edit that in this PR as well.

Reply:
I think uwp61 is probably the best label to use so we can be sure we don't clash with other packages.